packages/arb-validator-core/message/message.go

```diff
@@ -71,7 +77,86 @@ func intValueToAddress(val value.IntValue) common.Address {
 	return address
 }
 
-func UnmarshalUnsent(typecode MessageType, messageVal value.Value, chain common.Address) (UnsentMessage, error) {
+func addressesToValue(addresses []common.Address) value.TupleValue {
+	var ret []value.Value
+	for _, address := range addresses {
+		ret = append(ret, addressToIntValue(address))
+	}
+	return ListToStackValue(ret)
+}
+
+func valueToIntValuess(val value.Value) ([]value.IntValue, error) {
+	rawVals, err := StackValueToList(val)
+	if err != nil {
+		return nil, err
+	}
+	ints := make([]value.IntValue, 0, len(rawVals))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)